aibolit/utils/java_parser.py

```diff
@@ -41,111 +34,3 @@ def __init__(self, line, method_line, node, scope):
         self.method_line = method_line  # line number where parent method declared
         self.node = node  # javalang AST node object
         self.scope = scope  # ID of scope this node belongs
-
-
-class JavalangImproved:
```

Comment:
@KachanovYev thank you for the PR!
@yegor256 this PR removes unused classes `JavalangImproved` and `Lines`, which are closely related.